### ISSUE TITLE
Lost access to the balance in 2.5.2 (no connection issue) and in 2.5.3 the balance is 0

### ISSUE THREAD

**Issue by constantin-reea** [iotaledger/wallet#422]
I used the reclaim tool from the old address to the new address, but the balance is 0 although some days passed since installed the 2.5.3 wallet. 
What should I do? When can I expect my balance to recover the Iota coins?

**Comment by blutrumpet** [comment]
Same here, all my coins are gone :-( 👎

**Comment by wonkytonky** [comment]
try to generate  a lot of addresses.. until you see balance

**Comment by constantin-reea** [comment]
I had 1 address and 1 transfer with 119 MIota. Planned to buy more, but totally lost my interest after this issue with not seeing my balance. I have a Bitcoin wallet and it served me 5 years straight with no money going into the void. So how do I effectively recover my balance and with it my money? I invested only 50$ in this coin until now, but how about those who put thousands in it? :|

**Comment by wonkytonky** [comment]
try to generate a lot of addresses.. until you see balance (about as much as you created before snapshot)

**Comment by constantin-reea** [comment]
My bad! I installed the 5.2.3 wallet after 5.2.2 didn't connected to any node. 

I entered the old account has and seen the balance is 0. Got intrigued and seen your recovery tool and some blog posts. Thought I have to create a new hash (81 A-Z9 chars) and use the recovery tool to link the balance from the old account to the new one. After linking several times and seen the balance is still 0, I tried again to log in with the old account and attached an address to the tangle (which previously I didn't do). The balance is now back and all's good.

Sorry for the issue, but I got really annoyed and the above actions is what I understood I should do reading blog posts and spread articles.

Maybe the solution helps others in my situation (which made the same mistakes or understood the same).
